modules/ROOT/pages/tgw-add-arm.adoc

```diff
@@ -0,0 +1,258 @@
+
+[[create_resource_share]]
+== Create a Resource Share
+
+. Sign in to your AWS corporate account.
+. In Anypoint Platform, click the *Create resource share* link on the *Add transit gateway* page. 
++
+The link opens the AWS RAM console to the page for creating a resource share in the region you specified.
+. On AWS, on the *Create resource share* page:
++
+--
+.. Under *Description*, enter a descriptive name for the resource share in the *Name* field.
+.. Under *Resources - _optional_*, select *Transit Gateways* from the *Select resource type* menu and select the transit gateway resource to share.
++
+The transit gateway ID appears in the *Selected resources* field.
+.. Under *Principals - _optional_*, ensure that *Allow external accounts* is selected, 
+enter the MuleSoft AWS account ID that appears on the *Add transit gateway* page, and click *Add*.
++
+The AWS account number appears in the *Selected principals* field.
+.. Under *Tags*, add a tag if you want.
+.. Click *Create resource share*.
+.. Copy the following values for the resource share you just created: *ID* and *Owner*.
+--
+. In Anypoint Platform, in the *Add transit gateway* window: 
++
+--
+.. Click *Next*.
+.. Paste the values you copied from AWS in the *ID* and *Owner* fields:
++
+_update screenshot - styles changed_
++
+image::tgw-add-resource-share.png[Add transit gateway page: Resource share ID and owner ID]
+
+*** The resource share *ID* field contains alphanumeric characters (a-z, A-Z, 0-9) and hyphens (-).
+*** The resource share *Owner* field contains only numbers.
+
+.. Click *Add*.
+--
++
+The *Transit Gateways* page shows the progress for connecting to AWS and adding the transit gateway to Anypoint Platform.
+
+When the transit gateway addition succeeds, the *Transit Gateways* page displays the *Transit gateway added* message.
+The *Transit Gateways* page updates the owner and ID from AWS and the state as *Available*:
+
+_Update screenshot - title and contents changed_
+
+image::tgw-resource-share-available.png[Transit gateway added]
+
+If the transit gateway addition fails, see xref:tgw-troubleshoot.adoc[Troubleshoot Transit Gateway Attachments].
+
+
+
+[[attach_vpc_tgw]]
+== Attach a VPC to the Transit Gateway
+
+After the transit gateway attachment succeeds, you can attach your Anypoint VPC to enable apps to access the transit gateway.
+
+To attach a VPC to the transit gateway:
+
+. In Anypoint Platform, on the *Transit Gateways* page, click *Attach VPC*.
++
+The *Attach VPC* button is disabled if no VPCs are available to attach or if the transit gateway addition failed. 
+. From the *Select VPC* window, select the VPC to attach.
++
+_Add screenshot?_
++
+The window displays only the VPCs that are:
++
+--
+* In the same region as the transit gateway
+* Not already attached to a transit gateway
+* In Business Groups for which you have Cloudhub Network Administrator permissions
```

Comment:
Does "Business Groups" require the init. caps.?

Reply:
Nope. Fixed.